src/server.py

```diff
@@ -436,8 +437,15 @@ def run_seta_details_query():
         parts = d[0].split("'")
         jobtype.append([parts[1], parts[3], parts[5]])
 
+    alljobs = JOBSDATA.jobtype_query()
+
+    # we return low value job list as default, otherwise we return high value jobs
+    if option == 'low':
+        low_value_jobs = [low_value_job for low_value_job in alljobs if
+                          low_value_job not in jobtype]
+        jobtype = low_value_jobs
```

Comment:
the comment says low value jobs by default, but I don't see how.  we should special case if option == 'high'